tests/Unit/bootstrap.php

```diff
@@ -15,7 +15,8 @@
  */
 function load_original_functions_before_mocking() {
 	$originals = [
-		'rocket_get_constant' => require_once WP_ROCKET_PLUGIN_ROOT . 'inc/constants.php',
+		'rocket_get_constant'      => require_once WP_ROCKET_PLUGIN_ROOT . 'inc/constants.php',
+		'rocket_direct_filesystem' => require_once WP_ROCKET_PLUGIN_ROOT . 'inc/functions/files.php'
```

Comment:
@Tabrisrp We don't want to load the file here. Rather, we want a path to the file. It's then loaded within the `foreach` if the function doesn't already exist in memory.

Reply:
I don't understand, I replicated what you did before for `rocket_get_constant`, with `rocket_direct_filesystem `. Else I was getting an error `Cannot redeclare function`